package.json

```diff
@@ -17,7 +17,12 @@
   },
   "keywords": [
     "eslint",
-    "config"
+    "eslintconfig",
+    "config",
+    "brigade",
+    "javascript",
+    "react",
+    "stylesheet"
```

Comment:
What I meant here was "why list stylesheet?" It doesn't seem like a relevant keyword for this project.